neuralmonkey/config/builder.py

```diff
@@ -23,6 +23,7 @@ def __init__(self, string):
 
     def create(self):
         class_parts = self.clazz.split(".")
+
```

Comment:
Takže bych to viděl nějak takhle:

``` python
def __init__(self, string):
  self.class_config_name = string
  self.class_full_name = "neuralmonkey.{}".format(self.class_config_name)

  class_tokens = self.class_full_name.split(".")
  self.class_name = class_tokens[-1]
  self.module_name = ".".join(class_tokens[:-1])
```
A pak obohatit tu error hlášku o to full name

Ale možná je to příliš razantní. ;-)